src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-rxjs/no-implicit-any-catch.ts

```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertNoImplicitAnyCatch: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "rxjs/no-implicit-any-catch",
+            },
+        ],
```

Comment:
This is missing the rule arguments as stated on https://github.com/cartant/rxjs-tslint-rules/blob/master/source/rules/rxjsNoImplicitAnyCatchRule.ts

Reply:
Good catch, there was no mention of this parameter on the tslint rules description page. https://github.com/cartant/rxjs-tslint-rules. 
Will fix it.

Reply:
@KingDarBoja fixed